### DISCUSSION TITLE
only indexed 100KB content

### DISCUSSION THREAD

**Discussion by momocg** [dadoonet/fscrawler#1083] [Q&A] [answered]
**Describe the bug**

when the file exceed 100KB,  the elasticsearch only indexed 100KB. 

**Job Settings**

default settings. only change the elasticsearch url.

**Logs**

no error log

**Versions:**
 - Elasticsearch: [7.10.2]
 - Fscrawler [7.2.7]

**Comment by dadoonet** [comment, marked as answer]
Did you try to change this? https://fscrawler.readthedocs.io/en/latest/admin/fs/local-fs.html#extracted-characters

**Reply by dadoonet** [reply]
No news so I marked my answer as the answer :)
